### PR TITLE
Application note for working with C2PA provenance in TAMS

### PR DESCRIPTION
**Pull request by chalford**
# Details
Add an application note detailing an approach for working with C2PA provenance data in TAMS Flows.

# Pivotal Story (if relevant)
Story URL: N/A

# Submitter PR Checks
_(tick as appropriate)_

- [ ] PR completes task/fixes bug
- [ ] API version has been incremented if necessary
- [ ] ADR status has been updated, and ADR implementation has been recorded
- [x] Documentation updated (README, etc.)
- [ ] PR added to Pivotal story (if relevant)
- [ ] Follow-up stories added to Pivotal

# Reviewer PR Checks
_(tick as appropriate)_

- [ ] PR completes task/fixes bug
- [ ] Design makes sense, and fits with our current code base
- [ ] Code is easy to follow
- [ ] PR size is sensible
- [ ] Commit history is sensible and tidy

# Info on PRs
The checks above are guidelines. They don't all have to be ticked, but they should all have been considered.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -50,3 +50,4 @@ For more information on how we use application notes, see [here](./appnotes/READ
 | [0008](./appnotes/0008-timestamps-in-TAMS.md)                    | Timestamps in TAMS                                            |
 | [0009](./appnotes/0009-storage-label-format.md)                  | Storage label format specification                            |
 | [0010](./appnotes/0010-long-running-sources-and-flows.md)        | Long-running Sources and Flows                                |
+| [0010](./appnotes/0010-long-running-sources-and-flows.md)        | C2PA provenance across related Sources and Flows
```
Comment (j616) [inline docs/README.md:53]: Could you update the App Note number and link here?

**Comment by j616** [comment]
Looks like this isn't currently passing the markdown linter. You should be able to run this locally with `make lint` if you have docker installed

**Comment by chalford** [comment]
> Looks like this isn't currently passing the markdown linter. You should be able to run this locally with `make lint` if you have docker installed

Thanks. The remaining errors are:

* Inline HTML `a` element. This was to allow in-doc heading links/references. Is there a better way to do this, or should I replace with some text (e.g. see "Heading")
* Inline HTML center element. This was for the diagram captions. Markdown doesn't seem to have a solution here. Shall I just remove and let the caption left align?

**Review by j616 — APPROVED** [review]
Changes look good. Made a trivial change myself to fix a missed instance of the document number

**Pull request merged**